_data/simple-icons.json

```
@@ -4417,6 +4417,14 @@
             "hex": "4D148C",
             "source": "https://newsroom.fedex.com"
         },
+        {
+            "title": "Fediverse",
+            "hex": "111111",
+            "source": "https://commons.wikimedia.org/wiki/File:Fediverse_logo_proposal.svg",
```

```suggestion
            "source": "https://gitlab.com/ayush-sharma/to-the-fediverse/-/blob/c046ba7f3ea4c100d9c7538bb42593c245fdf532/static/images/icon.svg",
```
The official icon is available here :)

---

Can I ask where you got that link from? The proposal was made in 2018 by Eukombos. "fediverse.to" is NOT an official website.